android/app/src/main/java/com/reactnativenavigation/core/objects/Screen.java

```diff
@@ -34,6 +34,7 @@
     private static final String KEY_TOOL_BAR_STYLE = "navigatorStyle";
     private static final String KEY_STATUS_BAR_COLOR = "statusBarColor";
     private static final String KEY_TOOL_BAR_COLOR = "toolBarColor";
+    private static final String KEY_TOOL_BAR_HIDDEN = "toolBarHidden";
```

Comment:
Lets change `toolBarHidden` to `navBarHidden` which is the property we [already use](https://github.com/wix/react-native-navigation#style-object-format) in iOS